test/sandbox/test/inline_template_test.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
```

Comment:
Mind adding a test showing these in use with slots?

Reply:
Happy to add more tests, but would you mind clarifying what you're looking for? Do you want the template to use slots, or did you want to see another component use an "inline template component" as a slot?

Reply:
I was thinking of the latter, but the former would be useful too. Just to make sure things play together happily.

Reply:
Added a test in https://github.com/ViewComponent/view_component/pull/1556/commits/ab0c15b5b779db487a0a4e417260324bfca3de4f